app/views/stats/subscriptions.html.erb

```diff
@@ -17,6 +17,7 @@
       <%= range * 10 %> - <%= range * 10 + 9 %>
     </td>
     <td>
+      <% tags = tags.sort %>
```

Comment:
Hi @IshaGupta18, nice job but maybe you could do `<% tags.sort.each do |tag| %>` instead? So we can remove this redundant line? What do you think?